teams/arewewebyet.toml

```diff
@@ -0,0 +1,11 @@
+name = "arewewebyet"
```

Comment:
This should be a subteam of dev-tools like [this](https://github.com/rust-lang/team/blob/2d25539ba2b08252d629373cb40851d676939f7a/teams/cargo.toml#L2). cc @Manishearth